clc/cmd/exec_context.go

```diff
@@ -188,14 +207,52 @@ func (ec *ExecContext) ExecuteBlocking(ctx context.Context, hint string, f func(
 			}
 			return v, stop, nil
 		case <-timer.C:
-			if ec.isInteractive && s != nil {
+			if !quite {
 				// ignoring the error here
-				_ = s.Start()
+				_ = sp.Start()
 			}
 		}
 	}
 }
 
+func (ec *ExecContext) Wrap(f func() error) error {
+	tic := time.Now()
+	err := f()
+	toc := time.Now()
+	took := toc.Sub(tic)
```

Comment:
Instead of subtraction, we can use `time.Since(start)`.

Reply:
Good idea: https://github.com/hazelcast/hazelcast-commandline-client/pull/198/commits/ed3ed7dec14574328ae5774cc860e0dbabdb5c77